srcpkgs/senpai/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'senpai'
+pkgname=senpai
+version=0.0.20211223
+revision=1
+_gitrev=8091752a67781273944e7a79a803b7a671378313
```

Comment:
if they don't have releases, we can't accept it

Reply:
why ? some other packages use the git tree

Reply:
Yes, but it's against [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements). If software has releases, we can be sure they tested it and consider it to be stable at that point. If not, we have to do all the testing, patching and deciding at what point it's stable on our own which is a big effort for some packages.

Reply:
That makes sense. I'll reach out to the devs and see if they might consider a release.

Reply:
I received a reply on this a couple of days after the last comment here. The main developer said they would start doing releases but they haven't so far. I emailed again a few days ago but no response yet. I'm still going to update this in hopes that they will though (and to clear up the other issues).

Reply:
Got a reply on IRC from the dev.

`cc0: hi! still interested buy my development spare time is limited atm`

I can close this if needed, otherwise since there is a little interest I can keep it up.

Reply:
@paper42 Okay, the dev emailed me this morning and they just added a release! I've updated this PR with that.